pkg/datamodel/openai.go

```diff
@@ -1,10 +1,8 @@
-package schema
+package datamodel
```

Comment:
I think that we should keep `schema`, or even go with `modelrequest`. I think `datamodel` is too much generic here (at beginning it almost confused me here)

Reply:
I went with a super generic name admittedly - I'm OK with pkg/schema instead!

I assume this applies to the whole pkg/datamodel, not just openai.go, and will refactor it as such :)